### DISCUSSION TITLE
Can the Screen Be Rotated 90 or 270 Degrees?

### DISCUSSION THREAD

**Discussion by pppaapp** [dakanji/RefindPlus#152] [General]
### Suggested Feature

![](https://i.imgur.com/rgKCKPY.jpeg)

It breaks my neck every time I turn on my laptop.....

### Background Information

I don't know....

**Comment by dakanji** [comment]
Converted to Discussion as just a question asked.

Well, rotation is apparently technically possible but no plans to take this on at this time.

**Reply by pppaapp** [reply]
That's sad news...

**Reply by pppaapp** [reply]
There doesn't seem to be a bootloader that supports screen rotation.

If RefindPlus could support screen rotation, it would be a huge relief for people like me.

**Comment by dakanji** [comment]
OpenCore supports rotation and could be used to guide implementation.
Just not a priority item at this time.
Perhaps there is a firmware/BIOS setting you can change for this as it seems a bit unusual.

**Reply by pppaapp** [reply]
Unfortunately, I couldn't find a screen rotation feature in the BIOS....

I hope it supports screen rotation someday.

**Comment by dakanji** [comment]
Looked at your pic again and it looks basically like a tablet in a dock and not a laptop as such.
I don't think it is possible to rotate until the OS is loaded on such tablets.

**Reply by pppaapp** [reply]
It's not a tablet.

It's a laptop, very small 

![](https://ae01.alicdn.com/kf/S77b97a97086b45dbaf3e15c24073e6b6K/GPD-3-8-16-RAM-PC.jpg)

**Comment by dakanji** [comment]
Well, while it may have been integrated by the maker and is not user detachable, it is still a tablet in a keyboard dock. That's why it works as it does with such loaders.

**Reply by pppaapp** [reply]
I see.

So you don't plan on creating a feature?

**Comment by dakanji** [comment]
> don't plan on creating a feature?

No. Stated here: https://github.com/dakanji/RefindPlus/discussions/152#discussioncomment-4998366

You might want to ask the maker to allow this in the firmware. The display, as on every other tablet, currently defaults to portrait until the OS is loaded and orientation can be detected. I doubt they will change this but worth a try.

Good luck!

**Comment by SteveZMTstudios** [comment]
rEFInd Original developer in his web said that he may add the rotate feature.

I think all we need to do is rotate the icons and patterns, the text image of each character, as well as the keyboard operation logic and the way text/images are arranged (from horizontal to vertical) before launch (when editing refind themes etc), so that they look no different than if the screen were rotated, without having to change the original output.
